collections/Laud_lat/MS_Laud_Lat_21.xml

```diff
@@ -39,46 +39,159 @@
                   </msItem>
                </msContents>
                <physDesc>
                   <objectDesc form="codex">
                      <supportDesc material="perg">
                         <support>parchment</support>
+                        <extent>145 leaves
+                           <dimensions unit="mm" type="leaf">
+                              <height>255</height>
+                              <width>175</width>
+                           </dimensions></extent>
+                        <foliation>1-145 in 19th-century pencil.</foliation>
+                        <collation>1(8)-13(8), 14(8+1) (fol. 105 inserted at the beginning of the quire) (fols. 1-104, 105-113) | 15(8)-18(8) (fols. 114-144). Quires numbered in roman on the bottom centre of the final verso. Later-medieval leaf signatures in the upper margin.</collation>
+                        <condition>Water damage.</condition>
                      </supportDesc>
+                     <layoutDesc>
+                        <layout writtenLines="15" rulingMedium="plummet">Ruled in plummet. The main text in one central column of 15 widely-spaced lines, with double horizontal bounding lines, ruled space <dimensions unit="mm" type="ruled">
+                           <height>150</height>
+                           <width>47</width>
+                        </dimensions></layout>
+                        <layout>Separate ruling for up to 38 lines of gloss, typically in the outer and inner columns but often extending into the upper and lower margins to form a 'textus inclusus'. Width of the inner column c. 30 mm., of the outer column c. 55 mm., the overall width of the written area c. 140-5 mm.</layout>
+                        <layout>Gloss and text both written above top line.</layout>
+                     </layoutDesc>
                   </objectDesc>
+                  <handDesc>
+                     <handNote script="protogothic">Protogothic book and gloss script. The main text perhaps in two hands, changing at fol. 127v. Marginal and interlinear gloss mostly by one other hand, but with marginal gloss by two other hands in quire 15 (fols. 114-121) and quire 16 (fols. 122-129)</handNote>
+                  </handDesc>
+                  <decoDesc>
+                     <decoNote type="unfilled">Spaces left for initials, not filled in.</decoNote>
+                  </decoDesc>
+                  <bindingDesc>
+                     <binding notAfter="1639" notBefore="1638">
+                        <p>Laudian binding of calf over pasteboards, stamped with Laud's arms in gilt.</p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
                <history>
                   <origin>
-                     <origDate calendar="Gregorian" notAfter="1200" notBefore="1100">12th century</origDate>
+                     <origDate calendar="Gregorian" notAfter="1260" notBefore="1240">12th century (middle?)</origDate>
```

Comment:
Presumably the dates should be 1160 and 1140, unless it is actually 13th century.

Would it be more transparent to say '1140s or 1150s?' than 'middle?'

Reply:
well spotted! 
I prefer "middle" - I think in general that using numerical dates can give a misleading impression of precision and should be reserved for occasions when the evidence for dating is stronger than it is here